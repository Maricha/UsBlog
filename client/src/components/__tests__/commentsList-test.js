import React from 'react';
import { shallow } from 'enzyme'; 
import CommentsList from '../commentsList';

describe('CommentList Component', () => {
  it('renders correctly', () => {
    const component = shallow(<CommentsList id={1} classes={{test: "sds"}}/>);
    expect(component).toMatchSnapshot();
  })
});