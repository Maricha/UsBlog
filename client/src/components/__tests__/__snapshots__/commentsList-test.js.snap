// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentList Component renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WithStyles(CommentsList)
    classes={
      Object {
        "test": "sds",
      }
    }
    id={1}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "classes": Object {
        "header": "CommentsList-header-2",
        "test": "undefined sds",
        "wrapper": "CommentsList-wrapper-1",
      },
      "id": 1,
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "classes": Object {
          "header": "CommentsList-header-2",
          "test": "undefined sds",
          "wrapper": "CommentsList-wrapper-1",
        },
        "id": 1,
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
